src/main/java/com/marklogic/kafka/connect/DefaultDatabaseClientConfigBuilder.java

```diff
@@ -133,13 +126,8 @@ protected DatabaseClientConfig configureCustomSslConnection(DatabaseClientConfig
                 }
                 KeyManager[] key = keyManagerFactory.getKeyManagers();
                 try {
-                    if (tlsVersion != null && tlsVersion.trim().length() > 0) {
-                        sslContext = SSLContext.getInstance(tlsVersion);
-                    } else {
-                        sslContext = SSLContext.getInstance("TLSv1.2");
-                    }
+                    sslContext = SSLContext.getInstance(tlsVersion);
```

Comment:
Simplified this logic, as we have a default value of "TLSv1.2" in place already.